css/style.css

```diff
@@ -211,108 +351,246 @@ p {
+  @media (min-width: 768px) and (max-width: 991px) {
+    .about {
+      margin: 14.2rem 0 0 0;
+      flex-direction: row;
+      height: 86rem; } }
+  @media (min-width: 992px) {
+    .about {
+      margin: 14.2rem 0 0 0;
+      flex-direction: row;
+      height: 86rem; } }
+  .about__landing {
+    margin: 7.4rem 1.85rem 0 3.2rem;
+    padding: 2.4rem;
+    flex-basis: 40%; }
+    @media (max-width: 375px) {
+      .about__landing {
+        margin: 6.4rem 0 0 0;
+        max-width: 37.5;
+        width: 100%;
+        padding: 0 2.4rem; } }
+    @media (min-width: 768px) and (max-width: 991px) {
+      .about__landing {
+        margin: 14.2rem 1.85rem 0 3.2rem; } }
+    @media (min-width: 992px) {
+      .about__landing {
+        margin: 14.2rem 1.85rem 0 14.2rem; } }
 
-.skills__lang-list--iconcs {
-  width: 4.8rem;
-  height: 4.8rem;
-  border-radius: 50%;
-  border: 1px solid #fff;
-  background: url("/img/css3.svg") no-repeat center center;
-  background-size: cover; }
+.skills {
+  flex-basis: 60%;
+  margin: 2.2rem 0.2rem 0 1.85rem; }
+  @media (max-width: 375px) {
+    .skills {
+      margin: 0; } }
+  @media (min-width: 992px) {
+    .skills {
+      margin: 14.2rem 14.2rem 0 1.85rem; } }
+  @media (min-width: 768px) and (max-width: 991px) {
+    .skills {
+      margin: 14.2rem 0.2rem 0 1.85rem; } }
+  .skills__list {
+    list-style: none;
+    margin: 0 3.6rem; }
+    @media (max-width: 375px) {
+      .skills__list {
+        margin: 0 2.4rem; } }
+  .skills__list-item:not(:first-child) {
+    padding: 2.3rem 0;
+    border-bottom: 1px solid var(--color-border); }
+  .skills__lang-header {
+    display: flex;
+    justify-content: space-between;
+    align-items: center; }
+  .skills__lang-text {
+    color: #000;
+    font-size: 2rem;
+    font-weight: 500; }
+  .skills__lang-icon-wrapper {
+    margin-right: 3rem; }
+  .skills__lang-list {
+    margin-top: 1.8rem;
+    list-style: none;
+    padding: 1rem; }
+    @media (min-width: 992px) {
+      .skills__lang-list {
+        display: flex; } }
+  .skills__lang-list--item {
+    border-radius: 0.8rem;
+    background-color: #f7f7f9;
+    margin: 1.2rem 0; }
+    @media (max-width: 375px) {
+      .skills__lang-list--item {
+        width: 30.3rem; } }
+    @media (min-width: 992px) {
+      .skills__lang-list--item {
+        margin-right: 1.2rem;
+        width: 12.2rem; } }
+  .skills__lang-list--link {
+    display: flex;
+    align-items: center;
+    gap: 1.6rem;
+    padding: 2.1rem 1.2rem;
+    text-decoration: none;
+    cursor: pointer; }
+    @media (min-width: 992px) {
+      .skills__lang-list--link {
+        flex-direction: column; } }
+  .skills__lang-list--text {
+    font-size: 1.5rem;
+    font-weight: 500;
+    line-height: 2rem;
+    color: #253858; }
+  .skills__lang-list--iconjs {
+    width: 4.8rem;
+    height: 4.8rem;
+    border-radius: 50%;
+    border: 1px solid #fff;
+    background: url("/img/javascript.svg") no-repeat center center;
```

Comment:
- [ ] [REQUIRED] Please, try to use relative paths instead of root paths ♻️. Because root paths (paths that start with `/` are not recognized on many deployment servers like github pages 📄.